client/dashboard/sites/overview/index.tsx

```diff
@@ -78,7 +76,7 @@ function SiteOverview() {
 				<Sidebar
 					site={ site }
 					phpVersion={ phpVersion }
-					primaryDomain={ primaryDomain }
+					siteSlug={ siteSlug }
```

Comment:
Since we're passing "site" do we need to pass "siteSlug"?

Reply:
Fixed